plugins/follow_me/follow_me.h

```diff
@@ -0,0 +1,170 @@
+#pragma once
+
+#include <iostream>
+#include <cmath>
+#include <functional>
+
+namespace dronecore {
+
+class FollowMeImpl;
+
+/**
+ * @brief FollowMe plugin is used to make your drone to follow your Ground Control Station (Phone/Tab). <br>
+ *
+ * Application need to obtain current GCS position from underying platform,
+ * and feed to it FollowMe, which makes your drone to move.
+ * When GCS keeps moving to a new position, position updates are sent to the drone
+ * accordingly; this results in drone movement relative to that of GCS.
+ *
+ * Please refer https://docs.px4.io/en/flight_modes/follow_me.html for more information.
+ *
+ */
+class FollowMe
+{
+public:
+    /**
+     * @brief Constructor (internal use only).
+     * @param impl Private internal implementation.
+     */
+    explicit FollowMe(FollowMeImpl *impl);
+
+    /**
+     * @brief Destructor (internal use only).
+     */
+    ~FollowMe();
+
+    /**
+     * @brief Results of FollowMe operations
+     */
+    enum class Result {
+        SUCCESS = 0, /**< @brief %Request succeeded. */
+        NO_DEVICE, /**< @brief No device connected. */
+        CONNECTION_ERROR, /**< @brief %Connection error. */
+        BUSY, /**< @brief Vehicle busy. */
+        COMMAND_DENIED, /**< @brief Command denied. */
+        TIMEOUT, /**< @brief %Request timeout. */
+        NOT_ACTIVE, /**< @brief FollowMe is not activated yet */
+        UNKNOWN /**< @brief Unknown error. */
+    };
+
+    /**
+     * @brief FollowMe Configuration.
+     * Reference: https://docs.px4.io/en/advanced_config/parameter_reference.html#follow-target
+     */
+    struct Config {
+        /**
+         * @brief Side to follow target from.
+         */
+        enum class FollowDirection {
+            FRONT_RIGHT, /**< @brief Follow from front right. */
+            BEHIND, /**< @brief Follow from behind. */
+            FRONT, /**< @brief Follow from front. */
+            FRONT_LEFT, /**< @brief Follow from front left. */
+            NONE
+        };
+        // Default Configuration
+        static const float DEF_HEIGHT_M; /**< @brief Default height above home: 8.0 mts */
+        static const float DEF_FOLLOW_DIST_M; /**< @brief Default Follow distance: 8.0 mts */
+        static const FollowDirection DEF_FOLLOW_DIR; /**< @brief Default follow direction: BEHIND */
+        static const float DEF_RESPONSIVENSS; /**< @brief Default responsiveness to target: 0.5 */
+
+        // Min & Max ranges as per PX4 FollowMe Configuration.
+        static const float MIN_HEIGHT_M; /**< @brief Min follow height, in mts. >= 8.0m */
+        static const float MIN_FOLLOW_DIST_M;  /**< @brief Min follow distance from, in mts. >= 1.0m */
+        static const float MIN_RESPONSIVENESS; /**< @brief Min responsiveness: 0.0 */
+        static const float MAX_RESPONSIVENESS; /**< @brief Max responsiveness: 1.0 */
```

Comment:
You should be able to define the numbers right here using `static constexpr MAX_RESPONSIVENESS = 1.0f`.